lib/moon/components/switcher.ex

```diff
@@ -6,19 +6,16 @@ defmodule Moon.Components.Switcher do
   prop items, :list, default: []
   prop selected_item, :string
   prop click, :event, required: true
-  prop class, :string
+  prop class, :string, default: "flex items-center p-1 border rounded gap-x-2 border-beerus-100"
 
   def render(assigns) do
     ~F"""
-    <div
-      :if={length(@items) > 0}
-      class={"flex items-center p-1 border rounded gap-x-2 border-beerus-100 #{@class}"}
-    >
+    <div :if={length(@items) > 0} class={@class}>
       <button
         :for={item <- @items}
         class={
-          "px-2 py-1 text-moon-14 rounded",
-          "bg-piccolo-100 text-goten-100": item == @selected_item,
+          "p-1.5 lg:ps-2.5 lg:pe-3.5 font-medium text-sm rounded-md",
```

Comment:
What are ps and pe?

Reply:
basically just taken from moon.io site. do you think should be replaced with pl & pr?

Reply:
this is Rigth-To-Left paddings. in usual  (left-to-right) this is the same as pl and pr and for the Rioght-to-left reading they are pr and pl accordingly.